### PR TITLE
Use correct template path for production build (Issue 535)

### PR DESCRIPTION
**Pull request by arcos**
Fix for https://github.com/vttred/ose/issues/535

### CODE REVIEW

**Review by Rughalt — CHANGES_REQUESTED** [review]
Please remove code reformatting so it's easier to review+

**Comment by arcos** [comment]
> Please remove code reformatting so it's easier to review+

This formatting comes directly from the prettier setup in the codebase. So not sure what I should do here.

**Comment by arcos** [comment]
@Rughalt I reverted the formatting, but we need to have some kind of guideline on how code should be formatted. My IDE picked up the `.prettierrc` from the root of the project and started to auto format (based on my user setting in VS Code), if that's not how it should be done, what is the OSE standard?

**Comment by wyrmisis** [comment]
> @Rughalt I reverted the formatting, but we need to have some kind of guideline on how code should be formatted. My IDE picked up the `.prettierrc` from the root of the project and started to auto format (based on my user setting in VS Code), if that's not how it should be done, what is the OSE standard?

This is generally how it's been done. The only reason it didn't make it in pre-prettified is because my editor didn't already do this (which is a reminder to me to go look at why that is).

File: src/module/combat/combat-set-groups.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default class OSECombatGroupSelector extends HandlebarsApplicationMixin(A
 
   static PARTS = {
     main: {
-      template: `/systems/ose-dev/dist/templates/apps/combat-set-groups.hbs`
+      template: `/systems/${game.system.id}/dist/templates/apps/combat-set-groups.hbs`
```
Comment (wyrmisis) [inline src/module/combat/combat-set-groups.ts:44]: This app finds the template when you do this, right? I think the reason I did this initially was `game.system.id` not being available when this component (but I forgot to swing back around to it and figure something out for it).

**Review by wyrmisis — APPROVED** [review]
I'm fine with the changes, provided the combatant group app launches.

File: src/module/combat/combat-set-groups.ts (hunk 1 above)
Comment (arcos) [reply src/module/combat/combat-set-groups.ts:44]: No it doesn't! I thought for sure it would. Ok fixing this

**Comment by Rughalt** [comment]
@wyrmisis @arcos I sat down and thought about prettifying things, and I am all up for it if you do it in two commits in one PR - first, do prettify, and then do the fix. That way we can look at the fix separately from the prettify changes

**Comment by Rughalt** [comment]
I also think I want to fix it the other way; while changing the hardcoded path is good for production it breaks dev-env; what I will do is I will replace all `ose-dev` in the dist files with `ose` during package release procedure

**Comment by Rughalt** [comment]
I added the alternative approach in #537

**Comment by arcos** [comment]
@Rughalt I am going to close this PR, your solution is better.

**Pull request closed without merging**